ui/src/hooks/useTerminal.ts

```diff
@@ -40,7 +40,6 @@ import {
   wsIsActivated
 } from '@/components/TerminalComponent/helper';
 import mittBus from '@/utils/mittBus.ts';
-import { CollectionsBookmarkFilled } from '@vicons/material';
 
 enum MessageType {
   PING = 'PING',
```

Comment:
I'm sorry for the confusion but I need more context on which code you're referring to. Could you please specify the code snippet?